exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -163,6 +163,7 @@ public void start() throws Exception {
     embeddedJetty.addConnector(connector);
 
     final boolean portHunt = config.getBoolean(ExecConstants.HTTP_PORT_HUNT);
+    embeddedJetty.setDumpAfterStart(config.getBoolean(ExecConstants.HTTP_JETTY_SERVER_DUMP_AFTER_START));
```

Comment:
Could you please explain why? The method called already before the port hunting loop. Moving it below doesn't make sense since it will be too late to set the flag on ```embeddedJetty``` object.

Reply:
It is a minor comment. Placing it after declaring the `portHunt` variable and before the loop where port hunting happens enforces to think that it should be somehow connected with port hunting. Please move it above declaring the `portHunt` variable if possible.

Reply:
done.